react/features/base/config/reducer.ts

```diff
@@ -47,13 +47,11 @@ const INITIAL_RN_STATE: IConfig = {
     // FIXME: than requiring this override here...
 
     p2p: {
-        disabledCodec: 'vp9',
-        preferredCodec: 'h264'
+        mobileCodecPreferenceOrder: [ 'H264', 'VP8' ]
```

Comment:
What the value if we don't define this? We could make the code behave like so, and remove this override entirely, and people can configure whatever they want in config.js, no?

Reply:
https://github.com/jitsi/lib-jitsi-meet/pull/2296/files#diff-dcb81f0be0b559eee3d7693e162f67f90eee41d6fa0a8cbc91c6460ec02f5c36R38 will be the new default order which is applicable to desktop clients. We could define a different set for mobile endpoints by default.

Reply:
Let's do that please!